test/test_helper.rb

```diff
@@ -37,3 +39,12 @@ def request_host
 def available_testing_locales
   I18n.available_locales
 end
+
+
+WebMock.allow_net_connect!
+WebMock.stub_request(:any, "publiclab.org/api/srch/nearbyPeople")
+  .to_return(
+    body: "{items:[]}",
+    status: 200,
+    :headers => {"Content-Type"=> "application/json"}
+  )
```

Comment:
I think this worked!

Reply:
I think you can mock any testing framework now 😄 🎉 !!!